models/evc.py

```diff
@@ -374,6 +379,15 @@ def as_dict(self):
         evc_dict["flow_removed_at"] = self.flow_removed_at
         evc_dict["updated_at"] = self.updated_at
 
+        if keys:
```

Comment:
Let's also add a unit test for this, since essentially it's a new internal functionality.

Reply:
Unit test added in existing `test_as_dict()`.